source/modulo_controllers/include/modulo_controllers/BaseControllerInterface.hpp

```diff
@@ -919,4 +968,33 @@ inline void BaseControllerInterface::write_output(const std::string& name, const
   write_std_output<StringPublishers, std_msgs::msg::String, std::string>(name, data);
 }
 
+template<typename T>
+inline void BaseControllerInterface::publish_transforms(
+    const std::vector<state_representation::CartesianPose>& transforms, const std::shared_ptr<T>& tf_broadcaster,
+    bool is_static) {
+  if (this->get_node() == nullptr) {
+    throw modulo_core::exceptions::CoreException("Failed publish transforms: Node is not initialized yet.");
```

Comment:
```suggestion
    throw modulo_core::exceptions::CoreException("Failed send transform(s): Node is not initialized yet.");
```
I would probably do it like that since the public facing API is `send`